tests/auto_variables.py

```diff
@@ -0,0 +1,31 @@
+"""
+This module is used to simplify tests by adding global variables and parameters. 
+The idea is that you can use them everywhere without referencing:
+In test cases, fixtures, or parametrize.
+You can use the variables w, x, y, z, and the parameters a, b, c, d.
+"""
+import symfit as sf
+import pytest
+
+w, x, y, z = sf.variables('w, x, y, z')
+a, b, c, d = sf.parameters('a, b, c, d')
```

Comment:
Isn't it double to both add it here and as a fixture?

Reply:
Good catch ;)

Reply:
OTOH, you want to be able to import the variables/parameters...

Reply:
We have to use global variables (like them) because we can't use fixtures in Parametrizations. The fixture `initVariable` and `initParameter` reinitialize the globals after every test.